.pre-commit-hooks.yaml

```diff
@@ -0,0 +1,15 @@
+- id: datacontract-linting
```

Comment:
It would be a little more straight-forward to use `datacontract-lint` and `datacontract-test`, to use the same command names as in the cli. 
Or is there a convention to use the gerund?

Reply:
No, there is not specific rule behind it. We can give any id whatever we want. I agree, let's change to `datacontract-lint` and `datacontract-ltest` and people can use them as below:

```yaml
repos:
  - repo: https://github.com/datacontract/datacontract-cli
    rev: "v0.10.0"
    hooks:
      - id: datacontract-lint
      - id: datacontract-test
         args: ["--server", "http://example.com"]
```